examples/teaching-vm_openrefine/Dockerfile.original

```diff
@@ -0,0 +1,27 @@
+#docker build --rm -t psychemedia/ou-tm351-openrefine-test .
+#docker build --rm --build-arg BASE=etc -t psychemedia/ou-tm351-openrefine-test .
+
+ARG BASE=psychemedia/ou-tm351-minimal-test
+# Base machine
+FROM ${BASE}
+
+MAINTAINER tony.hirst@gmail.com
+
+ENV DOCKERBUILD 1
```

Comment:
What is DOCKERBUILD for?

Reply:
It's used in a script file that works both within a container and outside of it. It's explained in the README.